packages/react_rearch/lib/src/widgets.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:react/react.dart';
+import 'package:react/react_client.dart';
+import 'package:react_rearch/react_rearch.dart';
+import 'package:rearch/rearch.dart';
+
+part 'widgets/capsule_container_provider.dart';
+// part 'widgets/capsule_warm_up.dart';
```

Comment:
This file is currently commented out; would you mind just removing it? And maybe add in the RearchBootstrapper/similar that can take in the warmUpCapsules field